crates/object-store-util/src/lib.rs

```diff
@@ -149,6 +145,64 @@ pub async fn create_object_store_client(
     Ok(object_store)
 }
 
+/// Convert Restate [RetryPolicy] into [object_store::RetryConfig].
+fn from_retry_policy(retry_policy: &RetryPolicy) -> RetryConfig {
```

Comment:
Maybe this can be implemented as 
```
impl From<RetryPolicy> for RetryConfig{..}
```
it's more Rustic :)

Reply:
Ah i think this won't work if RetryPloicy from an external crate. Unless you implement this in the same module as RetryPolicy.

Reply:
Yeah, the orphan rule hit me here - and I didn't want to couple `RetryPolicy` to `object_store` as it's a core type of ours.